protostar/cheatable_starknet/cheatable_entry_point.py

```diff
@@ -280,3 +302,11 @@ def execute(
             general_config=general_config,
             tx_execution_context=tx_execution_context,
         )
+
+    def _warp_stark_exception(self, stark_exception: StarkException):
```

Comment:
```suggestion
    def _wrap_stark_exception(self, stark_exception: StarkException):
```

Reply:
[rename warp to wrap](https://github.com/software-mansion/protostar/pull/1404/commits/919ad296758202de5c5244c412aeea1488a7c3e4)